velox/common/base/RuntimeMetrics.h

```diff
@@ -18,11 +18,15 @@
 
 #include <fmt/format.h>
 #include <folly/CppAttributes.h>
+#include <folly/Range.h>
 #include <limits>
 #include <sstream>
 
 namespace facebook::velox {
 
+constexpr folly::StringPiece kJoinTableBuildWallNanos{
```

Comment:
do not use folly::StringPiece; use std::string or std::string_view

It is probably not a good idea to add dependency on velox/exec (join) to velox/common.

Can we move metric names to the module that produces them?

Reply:
Fixed